### PR TITLE
Reformat carbon folder

### PR DESCRIPTION
**Pull request by matthew7838**
### Summary
This should address all documentation issues in the `carbon_cycle` folder.

### CODE REVIEW

File: RUFAS/routines/field/soil/carbon_cycling/residue_partition.py [hunk 1]
```diff
@@ -2,30 +2,32 @@
...
+        provided.
+    field_size : float, optional
+        Used to initialize a SoilData object for this module to work with, if a pre-configured SoilData object is
+        not provided (ha)
+
+    Attributes
+    ----------
+    data : SoilData
+        The SoilData object used by this module to track residue in the soil profile.
+    References
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/carbon_cycling/residue_partition.py:25]:
```suggestion
    
    References
```

File: RUFAS/routines/field/soil/carbon_cycling/residue_partition.py [hunk 2]
```diff
@@ -60,7 +62,8 @@ def partition_residue(self, rainfall: float) -> None:
         Parameters
         ----------
         rainfall: float
-            amount of rain (mm)
+            amount of rain (mm).
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/carbon_cycling/residue_partition.py]:
```suggestion
            Amount of rain (mm).
```

File: RUFAS/routines/field/soil/carbon_cycling/residue_partition.py [hunk 3]
```diff
@@ -482,27 +503,29 @@ def _determine_plant_structural_to_slow_active_carbon_amount(
...
+            Rate at which above ground structural carbon decomposes into slow or active carbon (unitless).
         decomposition_moisture_effect: float
-            moisture effect on decomposition factor (unitless)
+            Moisture effect on decomposition factor (unitless).
         decomposition_temperature_effect: float
-            temperature effect on decomposition factor (unitless)
+            Temperature effect on decomposition factor (unitless).
         plant_structural_carbon_amount: float
-            pant structural carbon amount(kg/ha)
+            Pant structural carbon amount(kg/ha).
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/carbon_cycling/residue_partition.py]:
```suggestion
            Plant structural carbon amount (kg/ha).
```

File: RUFAS/routines/field/soil/carbon_cycling/residue_partition.py [hunk 4]
```diff
@@ -583,28 +611,30 @@ def _determine_plant_structural_carbon_amount(
...
 
         References
-        -------
+        ----------
         pseudocode_soil S.6.B.II.2
 
         Notes
-        -------
+        -----
         the referenced soil(below ground) biomass calculation was not found in the pseudocode_crop, neither is any
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/carbon_cycling/residue_partition.py]:
```suggestion
        The referenced soil(below ground) biomass calculation was not found in the pseudocode_crop, neither is any
```

File: RUFAS/routines/field/soil/carbon_cycling/pool_gas_partition.py [hunk 5]
```diff
@@ -405,9 +413,10 @@ def _determine_soil_active_carbon_amount(
         float
             Updated active carbon stored in the soil (kg/ha).
 
-        Notes
+        References
         -----
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/carbon_cycling/pool_gas_partition.py]:
```suggestion
        References
        ----------
```

File: RUFAS/routines/field/soil/carbon_cycling/carbon_cycle.py [hunk 6]
```diff
@@ -381,5 +419,6 @@ def _determine_total_carbon_CO2_lost(
         References
         -------
         pseudoode_soil S.6.D.7
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/carbon_cycling/carbon_cycle.py]:
```suggestion
        pseudocode_soil S.6.D.7
```

**Review by Isgotkowitz — APPROVED** [review]
Looks great, just spotted a few minor issues.

Nice job Matthew

**Review by allisterakun — APPROVED** [review]
LGTM

**Pull request merged**